### ISSUE TITLE
Fuzzy skin paint-on

### ISSUE THREAD

**Issue by Skivaks** [SoftFever/OrcaSlicer#4471]
### Is there an existing issue for this feature request?

- [x] I have searched the existing issues

### Is your feature request related to a problem?

Some parts i print get fuzzy skin where its not needed. Example below.
This is how the part looks like with Fuzzy skin off:
![fuzzy skin off](https://github.com/SoftFever/OrcaSlicer/assets/91622499/4c44a5ee-5751-4b8a-a70f-0bde4bb7e1df)
Here is how it looks with available options.
![fuzzy skin all walls](https://github.com/SoftFever/OrcaSlicer/assets/91622499/ff9fd5ab-62c3-4dcd-8280-ffdbc18e9b66)
![fuzzy skin and holes](https://github.com/SoftFever/OrcaSlicer/assets/91622499/edfde932-6a6a-4dfc-8b7f-a5bea9aed559)
![fuzzy skin contour](https://github.com/SoftFever/OrcaSlicer/assets/91622499/53dacc23-83db-4509-b496-4622496f08ea)

With this particular part i dont need the fuzzy skin on the post closer to the center of the model, only of the outside of the cap where i grab it.

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

Would be nice to have a Paint-on fuzzy skin tool like we have with Supports. Brushes - Circle, Sphere, Fill.

### Describe alternatives you've considered

I have tried using a Modifier to block Fuzzy skin generation, but it then generates Fuzzy skin where the blocker part ends. See screenshots below.
![fuzzy skin with blocker preview](https://github.com/SoftFever/OrcaSlicer/assets/91622499/ab15ca84-0f8a-4844-aeaa-c4159b8521a4)
![fuzzy skin with blocker sliced](https://github.com/SoftFever/OrcaSlicer/assets/91622499/5d1e393f-59e5-4de9-8744-bada511e9f53)

**Comment by DVSVIDEO** [comment]
♥In the first lines there is the line that you checked for existing problems...
This is a very old problem. Neither OrcaSlicer nor BambuSlicer existed when this bug was reported in Prusaslicer.
Numerous suggestions have been made (including by me) to dye the fuzzy skin...

**Comment by Skivaks** [comment]
> ♥In the first lines there is the line that you checked for existing problems...

I have only checked Issues on OrcaSlicer. I don't use Bambu or Prusa

**Comment by DVSVIDEO** [comment]
many-many problem in orcaslicer too
![fuzzy](https://github.com/SoftFever/OrcaSlicer/assets/96373457/eeb78f2b-3904-447f-80bf-76df32aa9b4a)

**Comment by PLAmenPRINT** [comment]
I have requested same thing more than year ago but unfortunately the idea was not supported enough.

https://github.com/SoftFever/OrcaSlicer/issues/269

I hope this option (fuzzy skin as paint) will be available some day :)
It will be very nice and usable.

**Comment by GeorgeStone25** [comment]
I would like to second the need for this feature. Fuzzy skin makes parts look so much better in general by disguising layer lines, but for people like me who make multi part functional prints, the fuzzy skin feature is unfortunately useless and no system that tries to guess where the fuzzy skin should apply is going to work. I need parts of prints to be smooth, and I need parts to have the fuzzy skin, or they don't work. 

We have paint on supports, so I am kind of surprised that paint on fuzzy skin isn't a thing. 

Even paint on where you just select a face, would be a massive improvement. That I imagine is much easier to do than free paint brush tool. 

I have also tried modifiers like OP, but it's so temperamental and a very frustrating way to do it. It often just adds fuzzy walls randomly in the middle of parts.

**Comment by jooslouw** [comment]
Up vote for this request. Paint-on will be great, but in most cases, selecting a face will be perfect.

Also, the problem with modifiers is that if you change your STEP or STL and reload, sometimes the modifiers are not aligned anymore, and you have to go through the painstaking process of realigning the modifiers. Painting or selecting faces will be much easier.

**Comment by DARKFiB3R** [comment]
This would be a very welcome feature

**Comment by simpat1zq** [comment]
I would love to see this feature.

**Comment by ph0tic** [comment]
you have no idea how insanely wanted/needed this is. injection molding does the same thing to hide imperfections.. this is the perfect way to do it with fdm

**Comment by Adenosina77** [comment]
Check this out: https://forum.bambulab.com/t/fuzzy-skin-all-or-nothing/5154/5

If you go to Objects, right click and "Add modifier" whatever suits you. Then you select this geometry and remove the fuzzy within. 

FINALLY I can print fuzzy wherever I want. 
I Had to share this with you.
![12312321](https://github.com/user-attachments/assets/a4ef8b9b-67b2-483c-ae29-de46c208db44)
![541342123](https://github.com/user-attachments/assets/b6eff63d-6aec-4924-b1d3-ca8db386a60d)

**Comment by ph0tic** [comment]
Nope don't want that. Creates an additional wall inside of the print for
the fuzzy skin. Really annoying. And then if you have a really curved
surface  good luck with simple modifiers. You can add a copy of the
original model as a modifier but with curves.. resizing correctly to hit it
just right and maybe not make an extra wall inside (sometimes works)  is
horriblly tedious.

So if you are making a square or a cylinder, great. If you are trying to
make something for ppl that abhor later lines...S O L

This is something that's been asked for from slizers for YEARS. I think
there's only one that can maybe do it (the one that you can imprint
designed on the faces).

Pls slicer gods, just make this happen!

**Comment by TylonHH** [comment]
> If it is solved in PrusaSlicer, or solved in SuperSlicer, it will be taken over by OrcaSlicer. But there, despite twice as many reports, no results. Either it is very difficult to solve this problem or the developers don't care....
> 
> read this: #2647

It's solved in prusa slicer with 

> You can select to apply Fuzzy skin only to **External perimeters** or to **All perimeters**.

https://help.prusa3d.com/article/fuzzy-skin_246186

**Comment by DARKFiB3R** [comment]
> > If it is solved in PrusaSlicer, or solved in SuperSlicer, it will be taken over by OrcaSlicer. But there, despite twice as many reports, no results. Either it is very difficult to solve this problem or the developers don't care....
> > 
> > read this: #2647
> 
> It's solved in prusa slicer with 
> 
> > You can select to apply Fuzzy skin only to **External perimeters** or to **All perimeters**.
> 
> https://help.prusa3d.com/article/fuzzy-skin_246186

That is not solved in the way we are requesting.

We want to be able to paint or fill specific areas with fuzzy skin, just like you can when painting colours.

**Comment by DBT85** [comment]
Adding my vote. Sometimes adding modifiers to complex geometries (especially since I can't find a way to change a parts opacity in the prepare view) where just being able to paint on fuzzy or block fuzzy would be really useful!

**Comment by DARKFiB3R** [comment]
I really hope this is being considered.

As DVSVIDEO said, maybe it's not as easy to impliment as us non-coders might imagine. I think the developers (at least 1?) do care though, how could they not? It would be such cool feature.

As would the ability to emboss an image/texture onto the surface of a model, like IdeaMaker's slicer.

**Comment by TylonHH** [comment]
BONUS Feature: instead of fuzzy skin also a svg with the texture would be a game changer. So you can draw any texture like leather, fur, zebra, waves, etc on the surface.

**Comment by amckimm** [comment]
Agreed with all of the above. Paint for many different functions: supports, fuzzy skin, textures, decals, text, embossing. Also ability to save paint template to transfer to other objects.

**Comment by simpat1zq** [comment]
Just a heads up that this was released in the PrusaSlicer Alpha 5 days ago. Also, I could be wrong but the PrisaSlicer Alpha release seems like it is saying that OrcaSlicer fixed the issue that OP mentioned in the first post regarding the modifiers printing fuzzy skin at the edge of the modifier block.

**Comment by ParasitKegel303** [comment]
+1 Porting this feature from PrusaSlicer would be great.

**Comment by TheOverpassArsonist** [comment]
> BONUS Feature: instead of fuzzy skin also a svg with the texture would be a game changer. So you can draw any texture like leather, fur, zebra, waves, etc on the surface.

I definitely agree (as far as I'm aware only ideamaker has this feature currently, which really sucks because it's not even just an aesthetic thing! You can add a pattern of ridges to your part to make it stiffer without adding more plastic, which means doing this is actually a great way to stop warping on larger parts!) but this should definitely be it's own issue.

also +1 on getting paint-on fuzzy skin ported

**Comment by TylonHH** [comment]
> also +1 on getting paint-on fuzzy skin ported

Then check this out: https://youtu.be/cNkHfydnUCI

**Comment by tlhintoq** [comment]
Fuzzy skin texture painting tool is clearly a desired item.  Just sayin'

https://github.com/SoftFever/OrcaSlicer/issues/7936
https://github.com/SoftFever/OrcaSlicer/issues/4471
https://github.com/SoftFever/OrcaSlicer/issues/3330

**Comment by FelipeRenault** [comment]
I've been using this for both top surface fuzzy skin and paint-on fuzzy skin:
https://github.com/TengerTechnologies/Fuzzyficator

There's a video about it too:
https://www.youtube.com/watch?v=cNkHfydnUCI

It's worth noting it still have the extra walls problem, though, since it's pretty much tricking the slicer into thinking it's a color change

**Comment by JimmyFallin** [comment]
I would also love to see the "Fuzzy Skin" paint option as trying to use modifiers sometimes is cumbersome and just doesn't work.

Thanks FelipeRenault for posting your solution to our request I am going to give this a shot.

**Comment by arekm** [comment]
Wouldn't "paint modifier" be a better, generic option - where area which you painted became a modifier of some depth? That would allow different modifications and not only fuzzy skin.
